### PR TITLE
Flutter driver patch: export finder factory

### PR DESCRIPTION
**Pull request by marcin-jelenski**
## Description

Patch: new mixin wasn't exported from flutter driver package.

## Related Issues

#67711

## Tests

N/A

## Checklist

Before you create this PR, confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I signed the [CLA].
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] All existing and new tests are passing.
- [X] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [X] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [X] No, no existing tests failed, so this is *not* a breaking change.

### CODE REVIEW

**Comment by marcin-jelenski** [comment]
@dnfield TBR. File export was missing. I've updated the statements in flutter driver. No tests were added, but if there's a way to check export correctness within framework, I'll extend.

**Review by dnfield — APPROVED** [review]

**Comment by dnfield** [comment]
Ideally we'd have a test in the repo covering this, but it's probably ok.

Just so you know - `TBR` is used by the Flutter team to mean `To Be Reviewed`, when someone is landing a patch without waiting for a review. We usually reserve it for a patch that needs to be landed to get the tree back to a consistent state, e.g. reverting a patch that unintentionally broke something :) We usually tag someone "to review it" at some point, because it still should get LGTM'd by someone with commit access even though it was already landed.

**Pull request merged**
